src/results.js

```diff
@@ -8,11 +8,12 @@ class TextResult extends Result {
    * @param {string} [text] text to be displayed
    * @param {int} [lineNum] line number of the result in the node
    */
-  constructor(text, yarnNodeData, lineNum) {
+  constructor(text, yarnNodeData, lineNum, formater) {
     super();
-    this.text = text;
+    this.text = text; // If needed to get the original text from, use the this.data w/ line number
     this.data = yarnNodeData;
     this.lineNum = lineNum;
+    this.formater = formater; // Formater Function used for the text
```

Comment:
why do we need to attach the formater here? Is this something bondagejs wrappers will ever make use of?

Reply:
If people want to use the formater, they can probably get to it in another way now. Remember when we attached it to the runner?
I'd say get rid of this - it will only make your life miserable on the equality checks in tests

Reply:
maybe if the user want's to uses it's own formater ? i can change it if it's not needed